tests/test_actions.py

```diff
@@ -464,7 +464,7 @@ def test_upload_data(user, id_token, status, preview, login_error, raises):
             "data_path": pathlib.Path.cwd() / "tests/test_data/upload/",
             "id": id_token,
             "preview": preview,
-            "target": "core",
+            "target": ["core"],
```

Comment:
This test was accidentally looking for any of `c`, `o`, `r`, or `e` in the file path.